.readme_generation/description.md

```diff
@@ -1,3 +1,5 @@
 <!-- Please provide a short overview of the features of this service. -->
 
-Here you should provide a short summary of the purpose of this microservice.
+The Notification Orchestration Service (NOS) uses data harvested from events published
+by various services to form Notification events, which are subsequently consumed by the
+Notification Service (NS) for dissemination.
```

Comment:
Just for posterity, we mentioned in the sprint review that the idempotence is now implemented in the NS. As a result of the same discussion, it is being considered whether we should merge the NS into the NOS (where the NOS sends the emails itself, or the functionality is somehow part of the NOS. Still requires the idempotence measure to prevent duplicate emails). Probably not going to go with the last suggestion.